api4/status_test.go

```diff
@@ -92,6 +92,18 @@ func TestGetUsersStatusesByIds(t *testing.T) {
 		assert.Equal(t, "dnd", userStatus.Status)
 	}
 
+	invalidUserIds := []string{"invalid_user_id", "invalid_user_id"}
+	_, resp = Client.GetUsersStatusesByIds(invalidUserIds)
+	CheckBadRequestStatus(t, resp)
+
+	invalidUserIds = []string{th.BasicUser.Id, "invalid_user_id"}
+	_, resp = Client.GetUsersStatusesByIds(invalidUserIds)
+	CheckBadRequestStatus(t, resp)
+
+	invalidUserIds = []string{"", ""}
+	_, resp = Client.GetUsersStatusesByIds(invalidUserIds)
+	CheckBadRequestStatus(t, resp)
```

Comment:
This is nice to have too but what I actually meant (sorry for not being super clear) was an empty slice `[]string{}` so we can test the first return case.

Reply:
was a bit confused by slice, added it

Reply:
No problem. That's the name Golang [uses](https://blog.golang.org/slices-intro) for that array-like data structure.